### ISSUE TITLE
 Generic type 'ModuleWithProviders<T>' requires 1 type argument(s). Angular 11

### ISSUE THREAD

**Issue by blasco** [angular/preboot#116]
* **I'm submitting a ...**
- [x] bug report
- [ ] feature request

* **Which parts of preboot are affected by this issue?**
- [ ] server side
- [ ] client side
- [ ] inline
- [x] build process
- [ ] docs
- [ ] tests

* **What is the current behavior?**
Outputs the following error:
```
 Generic type 'ModuleWithProviders<T>' requires 1 type argument(s).
```

**Comment by CaerusKaru** [comment]
Tracked in https://github.com/angular/preboot/pull/118

**Comment by blasco** [comment]
Great! Looking forward to it

**Comment by CaerusKaru** [comment]
Fixed in 8.0.0 release
